configs/platforms/fedora-f26-x86_64.rb

```diff
@@ -7,4 +7,5 @@
   plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/pl-build-tools-release-#{plat.get_os_name}-26.noarch.rpm"
   plat.install_build_dependencies_with "/usr/bin/dnf install -y --best --allowerasing"
   plat.vmpooler_template "fedora-26-x86_64"
+  plat.output_dir File.join("puppet5", "fedora", "f26", "x86_64")
```

Comment:
Do you want the 'f's? I would think not..

Reply:
Ah okay, I've removed the `f` from the path and left it in the package name